cpp/src/plasma/CMakeLists.txt

```diff
@@ -123,6 +129,18 @@ if ("${COMPILER_FAMILY}" STREQUAL "gcc")
     " -Wno-conversion")
 endif()
 
+if (PLASMA_BUILD_TENSORFLOW_OP)
+  if (NOT TensorFlow_FOUND)
+    message(FATAL_ERROR "Need TensorFlow to build the TensorFlow op")
+  endif()
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=0")
+  include_directories(${TensorFlow_INCLUDE_DIRS})
+  include_directories(${TensorFlow_INCLUDE_DIRS}/external/nsync/public)
```

Comment:
Hmm, (1) nsync is not included in the tf.sysconfig output?  (2) orthogonal to last question, is this needed here?  I never needed it to compile by hand.

Reply:
This is only needed for python 2.7 and yes, it's not included in the tf.sysconfig output. See https://github.com/sadeepj/crfasrnn_keras/issues/19 which I'm putting in.